src/tests/setupTests.ts

```diff
@@ -4,3 +4,7 @@ console.error = function (message: any, ...args: any[]) {
   error.call(console, message, ...args); // keep default behaviour
   throw message instanceof Error ? message : new Error(message);
 };
+
+console.warn = () => {
+  // ignore console warnings
```

Comment:
Not sure if warnings should be ignored.

Reply:
I was trying to get rid of the warning in the `calcite-color` test that uses an invalid value for setting a color. What do you think?

Reply:
I could update the test to intercept the warning message since it is expected.

Reply:
Ok. i'll remove the code for getting rid of all warnings :)

Reply:
OK, I'll update the test.